lib/esp_sdk/configure.rb

```diff
@@ -1,32 +1,36 @@
 module EspSdk
   class Configure
-    attr_accessor :token, :email, :version, :token_expires_at
+    attr_accessor :token, :email, :version, :token_expires_at, :end_point, :domain
 
     def initialize(options)
-      @email   = options[:email]
-      @version = options[:version] || 'v1'
-      token_setup(options)
+      self.email = options[:email]
```

Comment:
Not sure I understand why you went from @ to self. Its essentially the same thing.